test/e2e/frontend/cypress/tests/instances/editor.spec.js

```diff
@@ -85,7 +85,6 @@ describe('FlowForge - Instance editor', () => {
                 const instance = response.body.projects.find(
                     (project) => project.name === 'instance-1-1'
                 )
-                instance.name = 'qweqwe'
```

Comment:
😄

Reply:
qweqwe will be the death of me :sweat_smile:. I caught it in another pf and removed it

Reply:
Who's the `git blame` on that? Name and shame!

Reply:
That would be yours truly :raised_hand_with_fingers_splayed: :1st_place_medal: